yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -246,6 +246,58 @@ class TakeOperator : public FlowableOperator<T, T> {
 };
 
 template <typename T>
+class SkipOperator : public FlowableOperator<T, T> {
+ public:
+  SkipOperator(Reference<Flowable<T>> upstream, int64_t offset)
+      : FlowableOperator<T, T>(std::move(upstream)), offset_(offset) {}
+
+  void subscribe(Reference<Subscriber<T>> subscriber) override {
+    FlowableOperator<T, T>::upstream_->subscribe(
+        Reference<Subscription>(new Subscription(
+            Reference<Flowable<T>>(this), offset_, std::move(subscriber))));
+  }
+
+ private:
+  class Subscription : public FlowableOperator<T, T>::Subscription {
+    using Super = typename FlowableOperator<T,T>::Subscription;
+   public:
+    Subscription(
+        Reference<Flowable<T>> flowable,
+        int64_t offset,
+        Reference<Subscriber<T>> subscriber)
+        : FlowableOperator<T, T>::Subscription(
+             std::move(flowable),
+             std::move(subscriber)),
+         offset_(offset) {}
+
+    void onNext(T value) override {
+      if (offset_ <= 0) {
+        if (pending_ > 0) {
+          --pending_;
+          Super::subscriberOnNext(
+              std::move(value));
+        }
```

Comment:
Should we request further from the Super, up till we fulfill all the pending requests?